---
darknet_ros_msgs/BoundingBoxes result
